subsys/bluetooth/mesh/friend.c

```diff
@@ -340,33 +313,62 @@ static struct net_buf *create_friend_pdu(struct bt_mesh_friend *frnd,
 
 	net_buf_add_mem(buf, sdu->data, sdu->len);
 
-	/* We re-encrypt and obfuscate using the received IVI rather than
-	 * the normal TX IVI (which may be different) since the transport
-	 * layer nonce includes the IVI.
-	 */
-	if (bt_mesh_net_encrypt(enc, &buf->b, info->iv_index, false)) {
-		BT_ERR("Re-encrypting failed");
-		goto failed;
+	return buf;
+}
+
+static int encrypt_friend_pdu(struct bt_mesh_friend *frnd, struct net_buf *buf, bool master_cred)
+{
+	const u8_t *enc, *priv;
+	u8_t nid;
+
```

Comment:
No need for this empty line, i.e. you can group all variable declarations together. Not a very strongly enforced style rule, but what I find to be most readable, is the "inverted x-mas tree" style, i.e. longest declarations first, meaning the subnet would go on top.